src/scf-release/packages/acceptance-tests-brain/packaging

```diff
@@ -78,3 +78,7 @@ mv "${BOSH_COMPILE_TARGET}/github.com" "${BOSH_COMPILE_TARGET}/src/github.com"
 
 # We currently have no way of getting a valid version number, as we have no git repo
 GOBIN="${BIN_DIR}" go install -ldflags="-X main.version=0.0.0" github.com/SUSE/testbrain
+
+# Install the mysql client into the package
+zypper install -y mariadb-client
+cp /usr/bin/mysql ${BIN_DIR}
```

Comment:
How do we make sure we pull in the dependencies?

Reply:
The mysql client depends only on the system libraries like libc. No other dependencies.
Which is why I can get away with copying just that binary.